<?xml version="1.0" encoding="UTF-8"?><testsuite name="nosetests" tests="2" errors="2" failures="0" skip="0"><testcase classname="testtest.TestTaskControllers" name="test_sps_mock" time="0.004"><error type="exceptions.IOError" message="[Errno 2] No such file or directory: u'C:\\output\\tmp_result\\metatask-1428147743.19.json'&#10;-------------------- &gt;&gt; begin captured logging &lt;&lt; --------------------&#10;TaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.189000: Entering __init__(self=&lt;TaskController.BaseClass.Controller.Controller object at 0x04DDDA70&gt;, auto=False, priority=1000)&#10;TaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.189000: Exiting __init__(self=&lt;TaskController.BaseClass.Controller.Controller object at 0x04DDDA70&gt;, auto=False, priority=1000) returning: None&#10;TaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.190000: Entering add_metatask(self=&lt;TaskController.BaseClass.Controller.Controller object at 0x04DDDA70&gt;, path_or_dict={u'classes': {u'taskGenerator': u'TaskController.BaseClass.FullFactorialTGNoDymola.FullFactorialTGNoDymola', u'reporter': u'TaskController.BaseClass.BaseReporter.BaseReporter'}, u'taskGenerator': {u'taskGenerateName': u'TB Steadystate Generate', u'arguments': {u'variable': [{u'link': u'parameterOfFunction.testArguments.parameters', u'name': u'setRci', u'value': [u'0.1', u'0.6'], u'sdfName': u'RCI_P_cont'}, {u'displayUnit': u'rpm', u'name': u'setSpeed', u'value': u'500 1000', u'sdfName': u'w_P_cont', u'link': u'parameterOfFunction.testArguments.parameters', u'unit': u'rad/s', u'quantity': u'AngularVelocity'}, {u'displayUnit': u'degC', u'name': u'setTemp', u'value': [u'-10', u'40'], u'sdfName': u'T_P_cont', u'link': u'parameterOfFunction.testArguments.parameters', u'unit': u'K', u'quantity': u'Temperature'}], u'constant': {u'dependency': [u'EDRIS_wrapper.egg', u'BaseTask.py'], u'meta_function': u'SPSSystemMock.py', u'parameterOfFunction': {u'functionArguments': {u'setTorque': {u'errorRange': u'2', u'maxValue': u'1500', u'minValue': u'1'}}, u'testArguments': {u'modelName': u'EdrisLibComponents.SystemBases.SystemsBases.BasesSPS.TB_separated_SPS_base', u'simulationSettings': {u'StopTime': u'360000', u'Increment': u'0.5', u'Algorithm': u'8'}, u'results': [{u'sdfName': u't_P_cont', u'quantity': u'Time', u'unit': u's', u'signalName': u'Time'}, {u'sdfName': u'i_DC_P_cont', u'quantity': u'Current', u'unit': u'A', u'signalName': u'_signalBus_Main.batteryBus.general.current'}, {u'sdfName': u'i_ACRMS_P_cont', u'quantity': u'Current', u'unit': u'A', u'signalName': u'_signalBus_Main.electricMachine1Bus.general.currentRMS'}, {u'sdfName': u'v_P_cont', u'quantity': u'Voltage', u'unit': u'V', u'signalName': u'_signalBus_Main.batteryBus.general.voltage'}, {u'sdfName': u'P_cont', u'quantity': u'Power', u'unit': u'W', u'signalName': u'_signalBus_Main.electricMachine1Bus.general.powerMech'}, {u'sdfName': u'SOC_end_cont', u'signalName': u'_signalBus_Main.batteryBus.general.SOC'}, {u'sdfName': u'derating_cont', u'signalName': u'summarySystemSPS.deratingStatus'}], u'parameters': {u'timeOffset': u'0.2', u'setTorqueTolerance': u'0.5', u'RCIend_terminate': u'0.0001', u'HV_consumption': u'0'}}}}}}})&#10;TaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.191000: Entering __get_results(self=&lt;TaskController.BaseClass.Controller.Controller object at 0x04DDDA70&gt;)&#10;TaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.191000: Error at add_metatask: (Traceback (most recent call last):&#10;  File &quot;C:\Users\Lee\Downloads\modelon\EDRIS_Management_Tools_devel\dlog\__init__.py&quot;, line 62, in wrapped&#10;    f_result = fn(*v, **k)&#10;  File &quot;C:\Users\Lee\Downloads\modelon\EDRIS_Management_Tools_devel\TaskController\BaseClass\Controller.py&quot;, line 69, in add_metatask&#10;    with open(os.path.join(__conf__['outputPath'],'tmp_result', 'metatask-%s.json' %  time.time()), 'a') as f:&#10;IOError: [Errno 2] No such file or directory: u'C:\\output\\tmp_result\\metatask-1428147743.19.json'&#10;)&#10;--------------------- &gt;&gt; end captured logging &lt;&lt; ---------------------"><![CDATA[  File "C:\WinPython-32bit-2.7.6.4\python-2.7.6\lib\unittest\case.py", line 331, in run
    testMethod()
  File "C:\Users\Lee\Downloads\modelon\EDRIS_Management_Tools_devel\testtest.py", line 16, in test_sps_mock
    result = SSPS.start_controller(tid=100, input_data=None, controller=self.controller, block=False, method='SystemMock')
  File "C:\Users\Lee\Downloads\modelon\EDRIS_Management_Tools_devel\TaskController\SSPS\__init__.py", line 23, in start_controller
    result_future = controller.add_metatask(metatask_data)
  File "C:\Users\Lee\Downloads\modelon\EDRIS_Management_Tools_devel\dlog\__init__.py", line 65, in wrapped
    raise err
'[Errno 2] No such file or directory: u\'C:\\\\output\\\\tmp_result\\\\metatask-1428147743.19.json\'\n-------------------- >> begin captured logging << --------------------\nTaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.189000: Entering __init__(self=<TaskController.BaseClass.Controller.Controller object at 0x04DDDA70>, auto=False, priority=1000)\nTaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.189000: Exiting __init__(self=<TaskController.BaseClass.Controller.Controller object at 0x04DDDA70>, auto=False, priority=1000) returning: None\nTaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.190000: Entering add_metatask(self=<TaskController.BaseClass.Controller.Controller object at 0x04DDDA70>, path_or_dict={u\'classes\': {u\'taskGenerator\': u\'TaskController.BaseClass.FullFactorialTGNoDymola.FullFactorialTGNoDymola\', u\'reporter\': u\'TaskController.BaseClass.BaseReporter.BaseReporter\'}, u\'taskGenerator\': {u\'taskGenerateName\': u\'TB Steadystate Generate\', u\'arguments\': {u\'variable\': [{u\'link\': u\'parameterOfFunction.testArguments.parameters\', u\'name\': u\'setRci\', u\'value\': [u\'0.1\', u\'0.6\'], u\'sdfName\': u\'RCI_P_cont\'}, {u\'displayUnit\': u\'rpm\', u\'name\': u\'setSpeed\', u\'value\': u\'500 1000\', u\'sdfName\': u\'w_P_cont\', u\'link\': u\'parameterOfFunction.testArguments.parameters\', u\'unit\': u\'rad/s\', u\'quantity\': u\'AngularVelocity\'}, {u\'displayUnit\': u\'degC\', u\'name\': u\'setTemp\', u\'value\': [u\'-10\', u\'40\'], u\'sdfName\': u\'T_P_cont\', u\'link\': u\'parameterOfFunction.testArguments.parameters\', u\'unit\': u\'K\', u\'quantity\': u\'Temperature\'}], u\'constant\': {u\'dependency\': [u\'EDRIS_wrapper.egg\', u\'BaseTask.py\'], u\'meta_function\': u\'SPSSystemMock.py\', u\'parameterOfFunction\': {u\'functionArguments\': {u\'setTorque\': {u\'errorRange\': u\'2\', u\'maxValue\': u\'1500\', u\'minValue\': u\'1\'}}, u\'testArguments\': {u\'modelName\': u\'EdrisLibComponents.SystemBases.SystemsBases.BasesSPS.TB_separated_SPS_base\', u\'simulationSettings\': {u\'StopTime\': u\'360000\', u\'Increment\': u\'0.5\', u\'Algorithm\': u\'8\'}, u\'results\': [{u\'sdfName\': u\'t_P_cont\', u\'quantity\': u\'Time\', u\'unit\': u\'s\', u\'signalName\': u\'Time\'}, {u\'sdfName\': u\'i_DC_P_cont\', u\'quantity\': u\'Current\', u\'unit\': u\'A\', u\'signalName\': u\'_signalBus_Main.batteryBus.general.current\'}, {u\'sdfName\': u\'i_ACRMS_P_cont\', u\'quantity\': u\'Current\', u\'unit\': u\'A\', u\'signalName\': u\'_signalBus_Main.electricMachine1Bus.general.currentRMS\'}, {u\'sdfName\': u\'v_P_cont\', u\'quantity\': u\'Voltage\', u\'unit\': u\'V\', u\'signalName\': u\'_signalBus_Main.batteryBus.general.voltage\'}, {u\'sdfName\': u\'P_cont\', u\'quantity\': u\'Power\', u\'unit\': u\'W\', u\'signalName\': u\'_signalBus_Main.electricMachine1Bus.general.powerMech\'}, {u\'sdfName\': u\'SOC_end_cont\', u\'signalName\': u\'_signalBus_Main.batteryBus.general.SOC\'}, {u\'sdfName\': u\'derating_cont\', u\'signalName\': u\'summarySystemSPS.deratingStatus\'}], u\'parameters\': {u\'timeOffset\': u\'0.2\', u\'setTorqueTolerance\': u\'0.5\', u\'RCIend_terminate\': u\'0.0001\', u\'HV_consumption\': u\'0\'}}}}}}})\nTaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.191000: Entering __get_results(self=<TaskController.BaseClass.Controller.Controller object at 0x04DDDA70>)\nTaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.191000: Error at add_metatask: (Traceback (most recent call last):\n  File "C:\\Users\\Lee\\Downloads\\modelon\\EDRIS_Management_Tools_devel\\dlog\\__init__.py", line 62, in wrapped\n    f_result = fn(*v, **k)\n  File "C:\\Users\\Lee\\Downloads\\modelon\\EDRIS_Management_Tools_devel\\TaskController\\BaseClass\\Controller.py", line 69, in add_metatask\n    with open(os.path.join(__conf__[\'outputPath\'],\'tmp_result\', \'metatask-%s.json\' %  time.time()), \'a\') as f:\nIOError: [Errno 2] No such file or directory: u\'C:\\\\output\\\\tmp_result\\\\metatask-1428147743.19.json\'\n)\n--------------------- >> end captured logging << ---------------------']]></error></testcase><testcase classname="testtest.TestTaskControllers" name="test_sps_mock" time="0.007"><error type="exceptions.IOError" message="[Errno 2] No such file or directory: u'C:\\output\\tmp_result\\result-None.txt'&#10;-------------------- &gt;&gt; begin captured logging &lt;&lt; --------------------&#10;TaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.189000: Entering __init__(self=&lt;TaskController.BaseClass.Controller.Controller object at 0x04DDDA70&gt;, auto=False, priority=1000)&#10;TaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.189000: Exiting __init__(self=&lt;TaskController.BaseClass.Controller.Controller object at 0x04DDDA70&gt;, auto=False, priority=1000) returning: None&#10;TaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.190000: Entering add_metatask(self=&lt;TaskController.BaseClass.Controller.Controller object at 0x04DDDA70&gt;, path_or_dict={u'classes': {u'taskGenerator': u'TaskController.BaseClass.FullFactorialTGNoDymola.FullFactorialTGNoDymola', u'reporter': u'TaskController.BaseClass.BaseReporter.BaseReporter'}, u'taskGenerator': {u'taskGenerateName': u'TB Steadystate Generate', u'arguments': {u'variable': [{u'link': u'parameterOfFunction.testArguments.parameters', u'name': u'setRci', u'value': [u'0.1', u'0.6'], u'sdfName': u'RCI_P_cont'}, {u'displayUnit': u'rpm', u'name': u'setSpeed', u'value': u'500 1000', u'sdfName': u'w_P_cont', u'link': u'parameterOfFunction.testArguments.parameters', u'unit': u'rad/s', u'quantity': u'AngularVelocity'}, {u'displayUnit': u'degC', u'name': u'setTemp', u'value': [u'-10', u'40'], u'sdfName': u'T_P_cont', u'link': u'parameterOfFunction.testArguments.parameters', u'unit': u'K', u'quantity': u'Temperature'}], u'constant': {u'dependency': [u'EDRIS_wrapper.egg', u'BaseTask.py'], u'meta_function': u'SPSSystemMock.py', u'parameterOfFunction': {u'functionArguments': {u'setTorque': {u'errorRange': u'2', u'maxValue': u'1500', u'minValue': u'1'}}, u'testArguments': {u'modelName': u'EdrisLibComponents.SystemBases.SystemsBases.BasesSPS.TB_separated_SPS_base', u'simulationSettings': {u'StopTime': u'360000', u'Increment': u'0.5', u'Algorithm': u'8'}, u'results': [{u'sdfName': u't_P_cont', u'quantity': u'Time', u'unit': u's', u'signalName': u'Time'}, {u'sdfName': u'i_DC_P_cont', u'quantity': u'Current', u'unit': u'A', u'signalName': u'_signalBus_Main.batteryBus.general.current'}, {u'sdfName': u'i_ACRMS_P_cont', u'quantity': u'Current', u'unit': u'A', u'signalName': u'_signalBus_Main.electricMachine1Bus.general.currentRMS'}, {u'sdfName': u'v_P_cont', u'quantity': u'Voltage', u'unit': u'V', u'signalName': u'_signalBus_Main.batteryBus.general.voltage'}, {u'sdfName': u'P_cont', u'quantity': u'Power', u'unit': u'W', u'signalName': u'_signalBus_Main.electricMachine1Bus.general.powerMech'}, {u'sdfName': u'SOC_end_cont', u'signalName': u'_signalBus_Main.batteryBus.general.SOC'}, {u'sdfName': u'derating_cont', u'signalName': u'summarySystemSPS.deratingStatus'}], u'parameters': {u'timeOffset': u'0.2', u'setTorqueTolerance': u'0.5', u'RCIend_terminate': u'0.0001', u'HV_consumption': u'0'}}}}}}})&#10;TaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.191000: Entering __get_results(self=&lt;TaskController.BaseClass.Controller.Controller object at 0x04DDDA70&gt;)&#10;TaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.191000: Error at add_metatask: (Traceback (most recent call last):&#10;  File &quot;C:\Users\Lee\Downloads\modelon\EDRIS_Management_Tools_devel\dlog\__init__.py&quot;, line 62, in wrapped&#10;    f_result = fn(*v, **k)&#10;  File &quot;C:\Users\Lee\Downloads\modelon\EDRIS_Management_Tools_devel\TaskController\BaseClass\Controller.py&quot;, line 69, in add_metatask&#10;    with open(os.path.join(__conf__['outputPath'],'tmp_result', 'metatask-%s.json' %  time.time()), 'a') as f:&#10;IOError: [Errno 2] No such file or directory: u'C:\\output\\tmp_result\\metatask-1428147743.19.json'&#10;)&#10;TaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.195000: Entering stop(self=&lt;TaskController.BaseClass.Controller.Controller object at 0x04DDDA70&gt;, all_stop=False)&#10;TaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.195000: Error at stop: (Traceback (most recent call last):&#10;  File &quot;C:\Users\Lee\Downloads\modelon\EDRIS_Management_Tools_devel\dlog\__init__.py&quot;, line 62, in wrapped&#10;    f_result = fn(*v, **k)&#10;  File &quot;C:\Users\Lee\Downloads\modelon\EDRIS_Management_Tools_devel\TaskController\BaseClass\Controller.py&quot;, line 101, in stop&#10;    with open(os.path.join(__conf__['outputPath'],'tmp_result', 'result-%s.txt' %self.cid), 'a') as f:&#10;IOError: [Errno 2] No such file or directory: u'C:\\output\\tmp_result\\result-None.txt'&#10;)&#10;--------------------- &gt;&gt; end captured logging &lt;&lt; ---------------------"><![CDATA[  File "C:\WinPython-32bit-2.7.6.4\python-2.7.6\lib\unittest\case.py", line 360, in run
    self.tearDown()
  File "C:\Users\Lee\Downloads\modelon\EDRIS_Management_Tools_devel\testtest.py", line 24, in tearDown
    self.controller.stop()
  File "C:\Users\Lee\Downloads\modelon\EDRIS_Management_Tools_devel\dlog\__init__.py", line 65, in wrapped
    raise err
'[Errno 2] No such file or directory: u\'C:\\\\output\\\\tmp_result\\\\result-None.txt\'\n-------------------- >> begin captured logging << --------------------\nTaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.189000: Entering __init__(self=<TaskController.BaseClass.Controller.Controller object at 0x04DDDA70>, auto=False, priority=1000)\nTaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.189000: Exiting __init__(self=<TaskController.BaseClass.Controller.Controller object at 0x04DDDA70>, auto=False, priority=1000) returning: None\nTaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.190000: Entering add_metatask(self=<TaskController.BaseClass.Controller.Controller object at 0x04DDDA70>, path_or_dict={u\'classes\': {u\'taskGenerator\': u\'TaskController.BaseClass.FullFactorialTGNoDymola.FullFactorialTGNoDymola\', u\'reporter\': u\'TaskController.BaseClass.BaseReporter.BaseReporter\'}, u\'taskGenerator\': {u\'taskGenerateName\': u\'TB Steadystate Generate\', u\'arguments\': {u\'variable\': [{u\'link\': u\'parameterOfFunction.testArguments.parameters\', u\'name\': u\'setRci\', u\'value\': [u\'0.1\', u\'0.6\'], u\'sdfName\': u\'RCI_P_cont\'}, {u\'displayUnit\': u\'rpm\', u\'name\': u\'setSpeed\', u\'value\': u\'500 1000\', u\'sdfName\': u\'w_P_cont\', u\'link\': u\'parameterOfFunction.testArguments.parameters\', u\'unit\': u\'rad/s\', u\'quantity\': u\'AngularVelocity\'}, {u\'displayUnit\': u\'degC\', u\'name\': u\'setTemp\', u\'value\': [u\'-10\', u\'40\'], u\'sdfName\': u\'T_P_cont\', u\'link\': u\'parameterOfFunction.testArguments.parameters\', u\'unit\': u\'K\', u\'quantity\': u\'Temperature\'}], u\'constant\': {u\'dependency\': [u\'EDRIS_wrapper.egg\', u\'BaseTask.py\'], u\'meta_function\': u\'SPSSystemMock.py\', u\'parameterOfFunction\': {u\'functionArguments\': {u\'setTorque\': {u\'errorRange\': u\'2\', u\'maxValue\': u\'1500\', u\'minValue\': u\'1\'}}, u\'testArguments\': {u\'modelName\': u\'EdrisLibComponents.SystemBases.SystemsBases.BasesSPS.TB_separated_SPS_base\', u\'simulationSettings\': {u\'StopTime\': u\'360000\', u\'Increment\': u\'0.5\', u\'Algorithm\': u\'8\'}, u\'results\': [{u\'sdfName\': u\'t_P_cont\', u\'quantity\': u\'Time\', u\'unit\': u\'s\', u\'signalName\': u\'Time\'}, {u\'sdfName\': u\'i_DC_P_cont\', u\'quantity\': u\'Current\', u\'unit\': u\'A\', u\'signalName\': u\'_signalBus_Main.batteryBus.general.current\'}, {u\'sdfName\': u\'i_ACRMS_P_cont\', u\'quantity\': u\'Current\', u\'unit\': u\'A\', u\'signalName\': u\'_signalBus_Main.electricMachine1Bus.general.currentRMS\'}, {u\'sdfName\': u\'v_P_cont\', u\'quantity\': u\'Voltage\', u\'unit\': u\'V\', u\'signalName\': u\'_signalBus_Main.batteryBus.general.voltage\'}, {u\'sdfName\': u\'P_cont\', u\'quantity\': u\'Power\', u\'unit\': u\'W\', u\'signalName\': u\'_signalBus_Main.electricMachine1Bus.general.powerMech\'}, {u\'sdfName\': u\'SOC_end_cont\', u\'signalName\': u\'_signalBus_Main.batteryBus.general.SOC\'}, {u\'sdfName\': u\'derating_cont\', u\'signalName\': u\'summarySystemSPS.deratingStatus\'}], u\'parameters\': {u\'timeOffset\': u\'0.2\', u\'setTorqueTolerance\': u\'0.5\', u\'RCIend_terminate\': u\'0.0001\', u\'HV_consumption\': u\'0\'}}}}}}})\nTaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.191000: Entering __get_results(self=<TaskController.BaseClass.Controller.Controller object at 0x04DDDA70>)\nTaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.191000: Error at add_metatask: (Traceback (most recent call last):\n  File "C:\\Users\\Lee\\Downloads\\modelon\\EDRIS_Management_Tools_devel\\dlog\\__init__.py", line 62, in wrapped\n    f_result = fn(*v, **k)\n  File "C:\\Users\\Lee\\Downloads\\modelon\\EDRIS_Management_Tools_devel\\TaskController\\BaseClass\\Controller.py", line 69, in add_metatask\n    with open(os.path.join(__conf__[\'outputPath\'],\'tmp_result\', \'metatask-%s.json\' %  time.time()), \'a\') as f:\nIOError: [Errno 2] No such file or directory: u\'C:\\\\output\\\\tmp_result\\\\metatask-1428147743.19.json\'\n)\nTaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.195000: Entering stop(self=<TaskController.BaseClass.Controller.Controller object at 0x04DDDA70>, all_stop=False)\nTaskController.BaseClass.Controller: INFO: 2015-04-04 13:42:23.195000: Error at stop: (Traceback (most recent call last):\n  File "C:\\Users\\Lee\\Downloads\\modelon\\EDRIS_Management_Tools_devel\\dlog\\__init__.py", line 62, in wrapped\n    f_result = fn(*v, **k)\n  File "C:\\Users\\Lee\\Downloads\\modelon\\EDRIS_Management_Tools_devel\\TaskController\\BaseClass\\Controller.py", line 101, in stop\n    with open(os.path.join(__conf__[\'outputPath\'],\'tmp_result\', \'result-%s.txt\' %self.cid), \'a\') as f:\nIOError: [Errno 2] No such file or directory: u\'C:\\\\output\\\\tmp_result\\\\result-None.txt\'\n)\n--------------------- >> end captured logging << ---------------------']]></error></testcase></testsuite>